apps/server/src/modules/authorization/index.ts

```diff
@@ -1,5 +1,13 @@
-export * from './authorization.module';
-export * from './authorization.service';
-export * from './authorization-context.builder';
-export * from './types';
-export * from './feathers';
+export { AuthorizationModule } from './authorization.module';
```

Comment:
Ich schreibs mal hier, weil man nicht direkt einen Ordner kommentieren kann: So wie ich das Zielbild verstanden hab, wollen wir, dass alles, was zur Domain-Schicht gehört, im Ordner "domain" liegt, oder nicht? Dann müssten doch hier alle Files (außer Module-Files, Readme und index)  in den domain-Ordner, weil der AuthorizationService zur Domain-Schicht gehört. (Mal außen vorgelassen, dass er momentan noch die Repos kennt.)

Meine Vorstellung von der Ordner-Struktur wäre dann:
```
authorization
> domain
    > service
    > helper
    > error
    ...
> controller (zukünftiger Ordner für die API)
-  ....module.ts
-  README.md
-  index.ts
```

Oder man lässt den domain-Ordner halt erstmal weg, weil es wegen der Repos im Loader momentan ja eh nicht stimmt.

Reply:
Es gibt noch das Ticket BC - 5469

Reply:
Angepasst, hat allerdings erheblich die File Changes erhöht da es dependency issues nach sich gezogen hat. Rules folder musste verschoben werden.

Reply:
Hm... Interessehalber: Wodurch genau sind die Dependency Cycles aufgetreten? Dadurch, dass du den service in den domain-Ordner verschoben hast?